packages/flutter/lib/src/cupertino/activity_indicator.dart

```diff
@@ -86,8 +90,8 @@ class CupertinoActivityIndicator extends StatefulWidget {
 
   /// The iOS version style of activity indicator.
   ///
-  /// Defaults to [CupertinoActivityIndicatorIOSVersionStyle.iOS13].
-  // TODO(ctrysbita): Change default style to iOS14 after official release, https://github.com/flutter/flutter/issues/60047
+  /// Defaults to [CupertinoActivityIndicatorIOSVersionStyle.iOS14].
+  // TODO(ctrysbita): Deprecate after official release, https://github.com/flutter/flutter/issues/62521
```

Comment:
Instead of deprecating just the iOS13 value, should you deprecate this parameter now?  That way in the future we only need one follow up PR to remove this parameter.  Otherwise you need one to deprecate this and a second later one to remove it.

Reply:
Sure, the parameter should be deprecated now to make user use default value.